server/views/applications/pages/offending-history/any-previous-convictions.njk

```diff
@@ -2,6 +2,13 @@
 {% block questions %}
   <h1 class="govuk-heading-l">{{ page.questions.hasAnyPreviousConvictions.question }}</h1>
 
+  <div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon govuk-!-margin-top-1" aria-hidden="true">!</span>
```

Comment:
I suspect we don't want aria-hidden="true" here. My understanding is that that will hide the content from screen readers?

Reply:
Ah I see, sorry, that's just the exclamation mark!